www/js/equation.js

```diff
@@ -69,6 +91,16 @@ define([
                                 var newId = subModel.addNode(newNodeOptions);
                                 nodeList.push({ "id": newId, "variable":variable});
```

Comment:
there aren't tabs here, it has spaces for some reason. Please fix them to tabs using sublime when you are committing it next time. Thanks